### DISCUSSION TITLE
filtering

### DISCUSSION THREAD

**Discussion by KateFanina** [mockapi-io/docs#6] [Q&A] [answered]
Good morning! Is it possible to filter with restriction ne nin?

**Comment by nsmirn0v** [comment, marked as answer]
Hi @KateFanina, thanks for the question. Unfortunately, it is not possible at the moment. Can you give me an example of how you would use it?

**Reply by KateFanina** [reply]
Of course, get tasks where id not in (15, 16, 17), this is nin condition. Or not equals (ne),  it is like id != 15. 
For example get not completted tasks, when I know tasks with id (15, 16, 17) are completted.
